### ISSUE TITLE
Get-BinRoot broken

### ISSUE THREAD

**Issue by ferventcoder** [chocolatey/choco#144]

**Comment by Nilzor** [comment]
Why is it broken and why are you deprecating it? You don't provide a good alternative in the [docs](https://github.com/chocolatey/choco/wiki/CreatePackages#3-path-provided-by-the-get-binroot-helper---will-be-deprecated-later-closer-to-v1-but-okay-to-use-for-now)  - the other being that you *can* install it to `%ChocolateyInstall%\lib\mypackage`. So the package developers are basically left with either a soon-to-be-deprecated option or one that the *can* use if they feel like it.

Don't you want your users to decide a default installation folder? Should it really be up to the package developer? Right now the target folders of the packages are total chaos. Either:

 - c:\program files
 - c:\program files (x86)
 - c:\chocolate\lib(..)
 - Whatever Get-BinRoot outputs
 - c:\ProgramData (when did that become a standard?)
 - Something completely different

Basically finding the executable after an installation is often a frustrating job. Most packages are not god on telling the user where it installs.

**Comment by ferventcoder** [comment]
@Nilzor note that it is NOT deprecated yet, but we are pre deprecating it.

**Comment by ferventcoder** [comment]
Basically many packages needed a get-bin root won't need it anymore so the recommendation will be to install it to the package folder since it is now stable and doesn't change. We are planning to release a new better named function called something like `Get-ToolsRoot` where folks can use the same idea of get-binroot that they were using before.

**Comment by ferventcoder** [comment]
@Nilzor it's not broken now. It was broken but fixed in the commit - please see https://github.com/ferventcoder/choco/commit/44306efbe55b982d92c9c23ba3b44df01e6f242e for details.

**Comment by Nilzor** [comment]
Ok I see, thanks for the update. 

"the recommendation will be to install it to the package folder" . You should be clear on the recommandations to the packet devs and update [the docs](https://github.com/chocolatey/choco/wiki/CreatePackages#3-path-provided-by-the-get-binroot-helper---will-be-deprecated-later-closer-to-v1-but-okay-to-use-for-now)?

**Comment by ferventcoder** [comment]
@Nilzor added #339.
